my-react-app/src/config/api.js

```diff
@@ -0,0 +1,12 @@
+import { create } from 'apisauce';
+
+const api = create({
+  baseURL: process.env.REACT_APP_API_BASE_URL,
+  timeout: 5000
+});
+
+export function setHeader(token) {
```

Comment:
Rename this function to be more descriptive of its functionality. `setAuthorizationHeader` would be nice